examples/complete/outputs.tf

```
@@ -35,3 +35,12 @@ output "services" {
   description = "Map of services created and their attributes"
   value       = module.ecs.services
 }
+
+################################################################################
+# Application Load Balancer
+################################################################################
+
+output "alb_dns_name" {
+  description = "ALB endpoint DNS"
```

From https://github.com/terraform-aws-modules/terraform-aws-alb/blob/46ec74232f910a22c2756473d24ee96dc62e4d3e/outputs.tf#L21C17-L21C52

```suggestion
  description = "The DNS name of the load balancer"
```

---

done!